packages/bpk-component-rating/src/BpkRating.module.scss

```
@@ -22,127 +22,55 @@ $bpk-spacing-v2: true;
 
 .bpk-rating {
   display: flex;
+  flex-flow: row nowrap;
+  align-items: baseline;
 
-  &--vertical {
-    flex-direction: column;
-  }
-
-  &__component {
-    justify-content: center;
+  &--large {
     align-items: center;
-    border-radius: 50%;
-    color: $bpk-color-white;
   }
 
-  &--base-rating {
+  &__score {
     display: flex;
-    width: bpk-spacing-lg() * 2;
-    height: bpk-spacing-lg() * 2;
-  }
+    padding-right: bpk-spacing-sm() * 2;
+    justify-content: center;
+    align-items: baseline;
+    color: $bpk-color-sky-gray;
 
-  &--lg-rating {
-    display: flex;
-    width: bpk-spacing-xxl() + (bpk-spacing-sm() * 5);
-    height: bpk-spacing-xxl() + (bpk-spacing-sm() * 5);
+    @include bpk-rtl {
+      padding-left: bpk-spacing-sm() * 2;
```

As this is 8px we can use the `md` token here
```suggestion
    padding-right: bpk-spacing-md();
```